build-system/tasks/build-bento.js

```diff
@@ -132,6 +132,7 @@ async function buildBentoComponent(name, version, hasCss, options = {}) {
     return;
   }
   const componentsDir = `src/bento/components/${name}/${version}`;
+  mkdirSync(`${componentsDir}/dist`, {recursive: true});
```

Comment:
what is this for?

Reply:
There is a race condition between writing js and css files. If the css task wins an error can be thrown because the directory doesn't exist. It seemed cleanest to create the directory before running either build task.

Reply:
1. is there a corresponding `mkdir` you can remove from the js task?
2. since the fn is async anyway, it would be infinitesmally better to make it `await mkdir ...`